### DISCUSSION TITLE
New release supporting Tailwind v3.0

### DISCUSSION THREAD

**Discussion by arthurgeek** [themesberg/flowbite#35] [Q&A] [answered]
Hi!

Thanks for this project!

I'm aware of the [1.2.1 project issues](https://github.com/themesberg/flowbite/projects/1), and I see you already added support for it, but are there any plans to release an earlier new version, just to add Tailwind 3.0 support, before finishing the other planned 1.2.1 features?

**Comment by zoltanszogyenyi** [comment, marked as answer]
Hey @arthurgeek,

Flowbite 1.2.0 is already fully compatible with the new version 3 of Tailwind CSS. We've already updated the [Flowbite documentation](https://flowbite.com/docs/getting-started/introduction/) and we'll mention this in the introduction and quickstart page.

I opened that issue before the stable release since I still wasn't sure based on the alpha release whether it would have breaking changes for Flowbite or not.

You can upgrade to Tailwind CSS version 3 and keep using Flowbite as it is without problems.

If you do encounter difficulties, just let us know here and we'll get right on top of it.

Regarding the issue, we want to update the classes with the new features such as colored shadows, new pseudo selectors, and more.

**Comment by arthurgeek** [comment]
Hi @zoltanszogyenyi! Perfect, thanks :)
